cwltool/main.py

```diff
@@ -695,6 +697,18 @@ def main(argsl=None,  # type: List[str]
                 argsl = sys.argv[1:]
             args = arg_parser().parse_args(argsl)
 
+        # If On windows platform, A default Docker Container is Used if not explicitely provided by user
+        if onWindows() and not args.default_container:
+            args.default_container = "ubuntu"
```

Comment:
I tested conformance tests on Alpine(size 5mb) docker image. 2 types of test are failing 1. test that use `bin/bash` 2. Test that use `rev` command is giving wrong output(Need to find reason for it)
I found another image https://github.com/frol/docker-alpine-bash `alpine-bash` its size is 6 mb and it comes with bash. So after using this 2 tests are failing that use `rev` reverse command, others are passing. Should we add external containers in tests that use bin/bash or keep `alpine-bash` as default image. And if so what would be a good bash image, `alpine-bash` is light.